multiplatform-markdown-renderer/src/commonMain/kotlin/com/mikepenz/markdown/model/MarkdownPadding.kt

```diff
@@ -32,18 +42,22 @@ private class DefaultMarkdownPadding(
 @Composable
 fun markdownPadding(
     block: Dp = 2.dp,
-    list: Dp = 8.dp,
+    list: Dp = 4.dp,
+    listItemTop: Dp = 4.dp,
     listItemBottom: Dp = 4.dp,
-    indentList: Dp = 8.dp,
+    /** Deprecated, please use `listIndent` instead */
```

Comment:
No one will know this is deprecated. You have to deprecate the entire function and add ReplaceWith()

Reply:
I would, however as they have the same overloads that would conflict :/